utils/lib.sh

```diff
@@ -37,17 +37,6 @@ if [[ -z ${DIFF_CMD} ]]; then
     fi
 fi
 
-DOT_CONFIG="${DOT_CONFIG:-${REPO_ROOT}/.config.sh}"
-
-source_dot_config() {
-    if [[ ! -e "${DOT_CONFIG}" ]]; then
-        err_msg "configuration does not exists at: ${DOT_CONFIG}"
-        return 42
-    fi
-    # shellcheck disable=SC1090
-    source "${DOT_CONFIG}"
-}
-
```

Comment:
You are correct, we do not need this here.  Can we still leave this function in here in SearXNG? As I wrote in the other post, I also use this library in other projects.  I had also written that I want to modularize the library.  It is very costly for me if I would have to do this modularization once here in SearXNG and then again in the other projects.  I want to use the same files in all projects.  Later, when the modularization is done, we will no longer need certain modules here in SearXNG and then these things would finally disappear from SearXNG.  My time is limited and I would like to avoid to do the same task twice .. it would be a great help to me if this is OK for you .. give me some time, I have an eye on.

Translated with www.DeepL.com/Translator (free version)